go.mod

```diff
@@ -71,6 +71,9 @@ require (
 	gopkg.in/yaml.v3 v3.0.1
 )
 
-require github.com/docker/libnetwork v0.8.0-dev.2.0.20190625141545-5a177b73e316 // indirect
+require (
+	github.com/docker/libnetwork v0.8.0-dev.2.0.20190625141545-5a177b73e316 // indirect
+	github.com/varlink/go v0.4.0
```

Comment:
Where did this come from?

Reply:
I'm not sure and it looks like I've a dirty tree.  Let me try that again.

Reply:
Nice spot @mheon!  Something must have gone wonky with my first vendor.  Looking better now.